cabal.project

```diff
@@ -1,4 +1,4 @@
-index-state: 2020-11-15T00:00:00Z
+index-state: 2021-02-01T00:00:00Z
```

Comment:
@nc6 is this going to cause any problems?

Reply:
Not now we've bumped haskell.nix